libr/core/cmd_eval.c

```diff
@@ -176,7 +180,11 @@ static void nextpal(RCore *core, int mode) {
 		r_list_free (files);
 		R_FREE (home);
 	}
-	files = r_sys_dir (R2_DATDIR"/radare2/"R2_VERSION"/cons/");
+
+	const char *dirPrefix = r_sys_prefix (NULL);
```

Comment:
Same here. Or add a primitive to construct that path in a single place

Reply:
You mean something like a function called r_str_prefix() that works like r_str_home()?